include/GTestResults.h

```diff
@@ -32,69 +30,28 @@
  */
 class GTestResults {
 
-public:
-	//! The run status of the GTest.
-	enum STATUS {
-		RUN,		//!< The test was run.
-		//! \todo Investigate other run status of a GTest.
-		UNDEFINED	//!< Currently not sure what the other possibilities are.
-	};
-
-	//! The outcome of the running of the test.
-	enum OUTCOME {
-		PASSED,		//The test passed.
-		FAILED		//The test has at least one failure.
-	};
-
 protected:
-	QString name;	//!< The name of the GTest that corresponds to this result.
-	uint time;		//!< The time it took to run this test (or suite of tests).
 	QStringList failureMessages;	//!< The list of failure messages --unparsed.
-	STATUS status;	//!< The run status of the unit test \see GTestResults::STATUS
+    QHash<QString, QString> attributes;
```

Comment:
It might make sense to make this a QHash<QString, QVariant> to allow for values of non-string data types. I.e. If you have a count, you're not required to convert to a string and back.